core/src/main/java/org/elasticsearch/index/mapper/core/DateFieldMapper.java

```diff
@@ -548,4 +554,13 @@ protected void doXContentBody(XContentBuilder builder, boolean includeDefaults,
             }
         }
     }
+
+    private static FormatDateTimeFormatter getDateTimeFormatter(Settings indexSettings) {
```

Comment:
I think this is cleaner to have inline? it is very short and not used anywhere else.